web-client/src/views/CaseDetail/PetitionPaymentForm.tsx

```diff
@@ -6,172 +6,182 @@ import { sequences, state } from '@web-client/presenter/app.cerebral';
 import React from 'react';
 import classNames from 'classnames';
 
-export const PetitionPaymentForm = connect(
```

Comment:
I refactored this file as an experiment to reduce typescript errors (`validateSequence` and `updateSequence` were unrecognized).